src/app/core-ui/main/status/modal/help-modal/command.model.ts

```diff
@@ -0,0 +1,20 @@
+export class Command {
+
+  constructor(private type: number, private text: string, private time: string, private code?: number) { }
+
+  public getType() {
```

Comment:
All below model helper function is really required? as there is no manipulation in   object

Reply:
obviously it is required check console-modal.component.html

Reply:
you can directly  access any key of a model like if you want to get a type 
`list.getType() `
`list.type ?`

Reply:
ok make sense to me :)